packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -546,6 +549,78 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
       }
     }
 
+    // Expand/Collapse options
+    if (isExpandableGridModel(model) && model.isRowExpandable(modelRow)) {
+      // If there are grouped columns, then it is a rollup
+      // For rollups, the column number will be the depth minus one
+      let cellValue =
+        model.groupedColumns.length > 0
+          ? model.textForCell(model.depthForRow(modelRow) - 1, modelRow)
+          : model.textForCell(0, modelRow);
+
+      if (cellValue === '') {
+        cellValue = 'null';
+      }
+
+      const getRowOptionFormatted = (
+        command: string,
+        cellVal: string,
+        len: number
+      ) => {
+        let newCellVal = cellVal;
+        if (command.length + cellVal.length + 3 > len) {
+          newCellVal = `${cellVal.substring(0, len - command.length - 6)}...`;
+        }
+        return `${command} "${newCellVal}"`;
+      };
```

Comment:
This function should be a static function, it doesn't need to be declared within this method as all the methods it uses are passed into it.
`len` could also be an optional parameter, defaulting to `30` (since that seems to be the value that's passed in every time).
Instead of using three dots (`...`), use the ellipses character (`…`): https://www.compart.com/en/unicode/U+2026
That will just take up one char instead.
There is already a `GridRenderer.truncate(str, len)` method already (which uses the ellipses instead of three dots), so you could write this method something like:
```
static function getRowOptionFormatted(command: string, cellValue: string, len = 30) {
  return `${command} "${GridRenderer.truncate(cellValue, len - command.length - 3)}"`;
}
```